recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
     - nodejs >=14.0.0
     - pyct >=0.4.4
     - packaging
-    - param >=1.9.2
+    - param >=1.12.0
```

Comment:
`param` can be `>=1.9.2 ` in `host` https://github.com/holoviz/panel/blob/bdf0b25ad9da156877cd627ce60cb6351edd124b/setup.py#L185